numbers/numbers.go

```diff
@@ -196,3 +196,11 @@ func DigitsBaseTen(x int64) int {
 	}
 	return count
 }
+
+func Abs(x int) int {
```

Comment:
Perhaps AbsInt? math.Abs does the same already for floats. Also needs Abs(+ or - Inf) to equal +Inf as a special case and perhaps the ability to handle NaN.

Reply:
Changed it to AbsInt. Not sure how to implement the rest. Might need your help on that.